proposals/description/repay_fuse_bad_debt.ts

```diff
@@ -0,0 +1,96 @@
+import { ProposalDescription } from '@custom-types/types';
+
+/*const fip_x: ProposalDescription = {
+  title: 'Repay Fuse Bad Debt',
+  commands: [
+    {
+      target: 'fuseFixer',
+      values: '0',
+      method: 'repayAll()',
+      arguments: [],
+      description: 'Repay all bad debt in Fuse pools 8, 18, 27, 127, 144, 146, 156'
+    }
+  ],
+  description: 'Repay all bad debt in Fuse pools 8, 18, 27, 127, 144, 146, 156'
+};
+
+export default fip_x;*/
+
+const fip_x: ProposalDescription = {
+  title: 'Repay Fuse Bad Debt',
+  commands: [
+    {
+      target: 'pcvGuardianNew',
+      values: '0',
+      method: 'setSafeAddress(address)',
+      arguments: ['{fuseFixer}'],
+      description: 'Set FuseFixer as a safe address'
+    },
+    {
```

Comment:
imo all the repays should be taken out. The PCV Guardian can safely repay one by one. This proposal should simply be setSafeAddress on fuseFixed imo

Reply:
Sure, this was just for testing purposes.

Reply:
Done.